.vscode/settings.json

```diff
@@ -18,7 +18,7 @@
 
   "eslint.codeAction.disableRuleComment": { "location": "sameLine" },
   "eslint.workingDirectories": [
-    { "pattern": "./apps/*/" },
-    { "pattern": "./packages/*/" }
+    { "pattern": "./apps/*", "!cwd": true },
+    { "pattern": "./packages/*", "!cwd": true }
```

Comment:
The problem is back and this seems to solve it. If I touch `cwd` in the workspace's ESLint config, ESLint can no longer find the `tsconfig.json` files and rules based on type-checking don't work.